src/libs/NavigationUtils.ts

```diff
@@ -1,5 +1,7 @@
+import _ from 'lodash';
```

Comment:
```suggestion
import cloneDeep from 'lodash/cloneDeep';
```

Try yo never import directly from lodash. In the long run it should be possible to remove the full lodash from project, and import only some of its specific packages